Detectors/TPC/base/include/TPCBase/Constants.h

```diff
@@ -0,0 +1,34 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// @file   Constants.h
+/// @author David Rohr
+///
+
+#ifndef AliceO2_TPC_Constants_H
+#define AliceO2_TPC_Constants_H
+
+namespace o2 { namespace TPC {
+//   enum RocType {ISector=0, OSector=1};
```

Comment:
I guess this comment can be removed.

Why do we need a separate Constants class at all?

Reply:
The problem with the sector class is that it has a dependency chain that in the end inherits root headers, which I cannot use for the GPU tracking.
But I need the basic constants like number of sectors, number of rots, etc.
And I do not want to duplicate them in a second constants file for GPU tracking.

Reply:
Ok, understood.
This is fine with me. We will need to check what else to put there.